integrations/aws/.port/resources/port-app-config.yml

```diff
@@ -0,0 +1,42 @@
+resources:
+  - kind: AWS::Organizations::Account
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .Id
+          title: .Name
+          blueprint: '"aws_account"'
+
+  - kind: cloudResource
+    selector:
+      query: 'true'
+      resourceKinds:
+        - AWS::Lambda::Function
+        - AWS::RDS::DBInstance
+        - AWS::S3::Bucket
+        - AWS::IAM::User
+        - AWS::ECS::Cluster
+        - AWS::ECS::Service
+        - AWS::Logs::LogGroup
+        - AWS::DynamoDB::Table
+        - AWS::SQS::Queue
+        - AWS::SNS::Topic
+        - AWS::Cognito::IdentityPool
+        - AWS::CloudFormation::Stack
+
+    port:
+      entity:
+        mappings:
+          identifier: .Identifier
+          title: .Identifier
```

Comment:
will it work now that you removed the identifier field?

Reply:
Yes still working